sync-zoho-user/src/main/java/eu/europeana/keycloak/zoho/SyncZohoUserProvider.java

```diff
@@ -41,25 +40,26 @@ public class SyncZohoUserProvider implements RealmResourceProvider {
 
     private static final Logger LOG = Logger.getLogger(SyncZohoUserProvider.class);
 
-    public static final String SYNC_REPORT_STATUS_MESSAGE = "{\"text\":\" %s accounts in Zoho where compared against %s accounts in KeyCloak where %s accounts are shared and the affiliation for %s accounts was changed or established.\"}";
+     public static final String SYNC_REPORT_STATUS_MESSAGE = "{\"text\":\" %s accounts in Zoho where compared against %s accounts in KeyCloak where:  %s accounts are shared and %s contacts were added to Zoho. \n"
```

Comment:
String is too long, better replace it with textBlocks that should solve the problem of new line and will be much more clean

example - 

// ORIGINAL
String message = "A-143, 9th Floor, Sovereign Corporate Tower,\n" +
      "Sector-136, Noida,\n" +
      "Uttar Pradesh - 201305";

// BETTER : Using text blocks 
// gets rid of lots of the clutter
String message = """
    A-143, 9th Floor, Sovereign Corporate Tower,
    Sector-136, Noida,
    Uttar Pradesh - 201305""";